ihontaryk/homework_10/recreate_generators.py

```diff
@@ -0,0 +1,30 @@
+def custom_zip(*iterable):
+    iterators = [iter(i) for i in iterable]
+    try:
+        while True:
+            yield tuple([next(i) for i in iterators])
```

Comment:
yield tuple(next(i) for i in iterators)

Reply:
![image](https://github.com/user-attachments/assets/c0459a6c-13e3-4991-850c-61749f491810)